common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -664,10 +665,20 @@ def get_content_titles(self):
 
     @classmethod
     def definition_from_xml(cls, xml_object, system):
-        children = [
-            system.process_xml(etree.tostring(child)).scope_ids.usage_id
-            for child in xml_object.getchildren()
-        ]
+        children = []
+
+        for child in xml_object.getchildren():
+            try:
+                children.append(system.process_xml(etree.tostring(child)).scope_ids.usage_id)
+            except (XMLSyntaxError, AttributeError):
+                msg = (
+                    "Unable to load child when parsing Library Content Block. "
+                    "This can happen when a comment is manually added to the course export."
+                )
+                logger.error(msg)
```

Comment:
This isn't an error, it would be better to do what XBlock does: explicitly check for a comment node and skip it.  But this will rarely happen, so let's move ahead.